website/docs/modules/ROOT/pages/cli/installation-ide.adoc

```diff
@@ -65,6 +65,25 @@ folder with one of the Mill example projects. There are a wide range of example
 from hello-world to multi-module libraries to client-server web applications, and you can
 pick one most similar to what you are doing so you can hit the ground running working.
 
+=== Bash/Zsh Tab Completion
+
+You can set up Bash and Zsh tab completions by subsequently running the command:
+
+[source,console,subs="verbatim,attributes"]
+----
+> ./mill mill.tabcomplete/install
+----
+
```

Comment:
Here we need a line explaining what `install` does:
* copy-ing complete-script to a specific location ~/.cache/mill/download
* modifying shell config files ~/.bash_profile and ~/.zshrc